src/Microsoft.PowerShell.Security/security/CredentialCommands.cs

```diff
@@ -86,20 +100,18 @@ public GetCredentialCommand() : base()
         /// </summary>
         protected override void BeginProcessing()
         {
-            if (!string.IsNullOrEmpty(this.Message))
-            {
-                string caption = UtilsStrings.PromptForCredential_DefaultCaption;
-                string message = this.Message;
+            string caption = this.Title ?? UtilsStrings.PromptForCredential_DefaultCaption;
+            string message = this.Message ?? "Enter your credentials.";
+            string username = this.UserName ?? null;
```

Comment:
If you are not going to use a different value when `this.UserName` is null, then `this.UserName ?? null;` is not necessary anymore. Line 96 can continue to use _userName.